spec/elastic_apm/spies/mongo_spec.rb

```diff
@@ -4,14 +4,30 @@
 
 module ElasticAPM
   RSpec.describe 'Spy: MongoDB' do
+    around do |ex|
+      start_mongodb
+      ex.run
```

Comment:
This is where I would wrap in `begin` ... `ensure` to make sure we shut it down whatever happens

Reply:
Since it is in an around hook the `stop_mongodb` will also be called if an exception happens, 
see [around-hook-with-exception](https://relishapp.com/rspec/rspec-core/docs/hooks/around-hooks#an-around-hook-continues-to-run-even-if-the-example-throws-an-exception)